test/irb/test_debugger_integration.rb

```diff
@@ -51,6 +51,56 @@ def test_debug
       assert_match(/=>   2\| puts "hello"/, output)
     end
 
+    def test_debug_class_to_s_return_string
```

Comment:
This test case is not needed.
It is testing a common normal case that to_s is defined and it returns string. I think this case is already tested in another test case.

Reply:
I removed this test case.